.github/workflows/makefile.yml

```diff
@@ -0,0 +1,52 @@
+name: Makefile
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  makefile:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          components: rustfmt
+
+      - uses: actions/checkout@v3
+
+      - name: Install
+        run: make install
+
+      - name: Install required rust toolchain
+        run: rustup toolchain install nightly-2022-05-24
+
+      - name: Add rust-src to toolchain
+        run: rustup component add rust-src --toolchain nightly-2022-05-24-x86_64-unknown-linux-gnu
```

Comment:
Can / should the make scripts do this for us? If not, it should be mentioned in the readme to manually run these commands.

Reply:
Yes, those two easily.

`wasm-pack` is a bit more complicated, we don't want to download and run the installer again, if wasm-pack is already installed!

I was planning to do a second PR for all three, but I can move those two to the Makefile now!

Reply:
Yeah, I agree wasm-pack is more complex, and can be handled later.

Reply:
I moved those two to the Makefile for now.